.github/workflows/update-snapshot.yml

```diff
@@ -72,7 +72,7 @@ jobs:
             * Point at new share images URL.
 
       - name: Wait for Preview URL to be live
-        run: npx wait-on -t 120000 https://covid-projections-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/compare/ || echo 'Preview URL failed to load.'
+        run: npx wait-on -t 120000 https://covid-act-now-website-git-bump-snapshot-${{env.RUN_ID}}-covidactnow.now.sh/compare/ || echo 'Preview URL failed to load.'
```

Comment:
@mikelehen Sean and I aren't 100% sure whether preview links such as this are generated with the beginning of the URL being our org or repo name. We assume the standard should be the repo name. Do you have any insight on this?

Reply:
Looks like the preview URL for this PR is: https://covid-projections-git-faihegberg-update-urls-8976ae-covidactnow.vercel.app/?s=44733217

So I think it must be the vercel project name and we shouldn't change it (unless we also update the vercel name).